src/services/goToDefinition.ts

```diff
@@ -510,4 +522,21 @@ namespace ts.GoToDefinition {
                 return false;
         }
     }
+
+    function createDefinitionInfoFromNode(node: Node, name: string): DefinitionInfo {
+        const sourceFile = node.getSourceFile();
+        return {
+            fileName: sourceFile.fileName,
+            textSpan: createTextSpanFromNode(node, sourceFile),
+            kind: ScriptElementKind.label,
+            name,
+            containerKind: ScriptElementKind.unknown,

```

Comment:
`undefined! ` just like in `createDefinitionInfoFromName`

Reply:
Should `containerKind` just be optional or something?